pkg/brew/ripgrep.rb

```diff
@@ -1,15 +1,15 @@
 require  'formula'
 class Ripgrep < Formula
-  version '0.1.15'
+  version '0.1.17'
   desc "Search tool like grep and The Silver Searcher."
   homepage "https://github.com/BurntSushi/ripgrep"
 
   if Hardware::CPU.is_64_bit?
     url "https://github.com/BurntSushi/ripgrep/releases/download/#{version}/ripgrep-#{version}-x86_64-apple-darwin.tar.gz"
-    sha256 "fc138cd57b533bd65739f3f695322e483fe648736358d853ddb9bcd26d84fdc5"
+    sha256 "cb7b551a08849cef6ef8f17229224f094299692981976a3c5873c93f68c8fa1a"
   else
     url "https://github.com/BurntSushi/ripgrep/releases/download/#{version}/ripgrep-#{version}-i686-apple-darwin.tar.gz"
-    sha256 "3ce1f12e49a463bc9dd4cfe2537aa9989a0dc81f7aa6f959ee0d0d82b5f768cb"
+    sha256 "0srpzhs943h0f66pvrbijdgalfrl33zgirv6alf6dzgkp5s6i4qf"
```

Comment:
Hmm, I think this should be `0e936874b9f3fd661c5566e7f8fe18343baa5e9371e57d8d71000e9234fc376b`.